### ISSUE TITLE
[BUG] incorrectly removing trailing `pass`.

### ISSUE THREAD

**Issue by sunmy2019** [hadialqattan/pycln#112]
**To Reproduce** Steps to reproduce the behavior:

1. Take this code snippet:

```python
import random
import time


def foo():
    pass


if random.randint(0, 1):
    foo()
    if random.randint(0, 1):
        foo()
else:
    pass

 ```

2. Run this Pycln command:
  ```bash
   $ python3.8 -m pycln test.py
   ```

3. Unexpected fixed code (if present):

```python
import random


def foo():
    pass


if random.randint(0, 1):
    foo()
    if random.randint(0, 1):
        foo()
else:

```

**Expected behavior**:

The generated python code should be legal.

**Environment (please complete the following informations):**

- Python Version: python3.8
- Pycln Version: pycln, version 1.1.0
- OS Type: Linux

**Comment by sunmy2019** [comment]
Duplicated https://github.com/hadialqattan/pycln/issues/99

**Comment by hadialqattan** [comment]
This already has been solved in v1.2.2
